frontend/src/features/rollout-announcement/components/ProgressIndicator.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box } from '@chakra-ui/react'
+import { motion } from 'framer-motion'
+
+const ActiveIndicator = (): JSX.Element => (
+  <Box
+    width="1.5rem"
+    height="0.5rem"
+    borderRadius="full"
+    backgroundColor="#5D6785"
+  />
+)
+
+interface InactiveIndicatorProps {
+  onClick?: () => void
+  isActiveIndicator: boolean
+}
+
+const CircleIndicator = (props: InactiveIndicatorProps): JSX.Element => {
+  const { onClick, isActiveIndicator } = props
+  return (
+    <Box
+      width="0.5rem"
+      height="0.5rem"
+      borderRadius="full"
+      backgroundColor="#BABECB"
+      marginRight={isActiveIndicator ? '1.5rem' : '0.5rem'}
+      onClick={onClick}
+    />
+  )
+}
+
+interface ProgressIndicatorProps {
+  numIndicators: number
+  currActiveIdx: number
+  onClick?: (indicatorIdx: number) => void
+}
+
+export const ProgressIndicator = (
+  props: ProgressIndicatorProps,
+): JSX.Element => {
+  const { numIndicators, currActiveIdx, onClick } = props
```

Comment:
can destructure this in the function arguments
```suggestion
  { numIndicators, currActiveIdx, onClick }: ProgressIndicatorProps,
): JSX.Element => {
```

Reply:
Okay updated the PR to do this, thanks!